snap/export_test.go

```diff
@@ -22,4 +22,5 @@ package snap
 var (
 	ImplicitSlotsForTests        = implicitSlots
 	ImplicitClassicSlotsForTests = implicitClassicSlots
+	RelativeHooksDir             = relativeHooksDir
```

Comment:
I don't understand this, what it buys us is that I can add a typo in relativeHooksDir and the test might still pass